package.json

```diff
@@ -9,6 +9,7 @@
   ],
   "scripts": {
     "compile": "hardhat compile",
+    "compile:harnesses": "env SRC=./certora/harnesses FOUNDRY=false npm run compile",
```

Comment:
Is `FOUNDRY=false` absolutelly necessary here ? Is the foundry package actually preventing compilation?

Reply:
Yes, otherwise we get this error:

```shell
> openzeppelin-solidity@5.0.1 compile:harnesses
> env SRC=./certora/harnesses npm run compile


> openzeppelin-solidity@5.0.1 compile
> hardhat compile

Error in plugin hardhat-foundry: User-configured sources path (./certora/harnesses) doesn't match path configured in foundry (contracts)
```

It's just a path mismatch or something similar. The fix would be to make the `foundry.toml` to match I think, but that's more work.